.travis.yml

```
@@ -1,9 +1,7 @@
 ---
 language: node_js
 node_js:
-  # we recommend testing addons with the same minimum supported node version as Ember CLI
-  # so that your addon works for all apps
-  - "6"
+  - "10"
```

If you always want to get the latest LTS without having to update `.travis.yml` every October, use this:

```suggestion
  - "lts/*"
```

---

I'd still need to update `package.json` though, unless there is a notation I can use there as well?